WebAPI/CurrencyConverter/Interfaces/IUnitOfWork.cs

```diff
@@ -0,0 +1,9 @@
+﻿using CurrencyConverter.Data;
+
+namespace CurrencyConverter.Interfaces
+{
+    public interface IUnitOfWork
+    {
+        ExchangeRateRepository ExchangeRateRepo { get; }
```

Comment:
We can add method Confirm and when implementig this method we can call _context.SaveChanges(). By doing this we can remove Save method for all IRepository and reduce redundant code.